light-curve/light_curve/light_curve_py/features/rainbow/temperature.py

```diff
@@ -0,0 +1,176 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Dict, List
+
+import numpy as np
+
+__all__ = ["temperature_terms", "BaseTemperatureTerm", "ConstantTemperatureTerm", "SigmoidTemperatureTerm"]
+
+
+@dataclass()
+class BaseTemperatureTerm:
+    """Bolometric term for the Rainbow"""
+
+    @staticmethod
+    @abstractmethod
+    def parameter_names() -> List[str]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def parameter_scalings() -> List[str]:
+        """Describes how to unscale the parameters - like time, timescale, flux or do not scale"""
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def value(self, t, params) -> float:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def initial_guesses(t, m, sigma, band) -> Dict[str, float]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def limits(t, m, sigma, band) -> Dict[str, float]:
+        return NotImplementedError
+
+
+@dataclass
+class ConstantTemperatureTerm(BaseTemperatureTerm):
+    """Constant temperature"""
+
+    @staticmethod
+    def parameter_names():
+        return ["T"]
+
+    @staticmethod
+    def parameter_scalings():
+        return [None]
```

Comment:
We use `None`, but base class says it should be `str`. Could you please either change the type hint (here and in `bolometric.py`) or always use strings?

Reply:
done